hedera-node/src/main/resources/bootstrap.properties

```diff
@@ -78,7 +78,7 @@ tokens.maxSymbolUtf8Bytes=100
 tokens.maxTokenNameUtf8Bytes=100
 tokens.maxCustomFeesAllowed=10
 tokens.maxCustomFeeDepth=2
-tokens.nfts.areEnabled=false
+tokens.nfts.areEnabled=true
```

Comment:
Now that we're in the `0.17.0` development iteration, good idea!

But CI currently begins with [a spec](https://github.com/hashgraph/hedera-services/blob/master/test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java#L118) that requires HIP-17 to be _off by default_; so we need to change that spec while setting the default to be `tokens.nfts.areEnabled=true`.

(Maybe the spec can now begin by toggling HIP-17 off or something.)

Reply:
I've switched that back to tokens.nfts.areEnabled=false till the spec changes